src/main/java/nextstep/subway/ui/LineController.java

```diff
@@ -0,0 +1,55 @@
+package nextstep.subway.ui;
+
+import nextstep.subway.application.LineService;
+import nextstep.subway.dto.LineRequest;
+import nextstep.subway.dto.LineResponse;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+@RestController
```

Comment:
@RequestMapping을 활용해볼 수 있겠네요
```suggestion
@RestController
@RequestMapping("/lines")
```

Reply:
네~ 감사합니다. 
```text
@RequestMapping(value = "/lines", produces = MediaType.APPLICATION_JSON_VALUE)
```
조언 주신것 처럼 공통된 부분을 처리되도록 수정하였습니다 ^^ 👍